dev/config/apiservice.yaml

```diff
@@ -0,0 +1,19 @@
+
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1.appcat.vshn.io
+  labels:
+    api: appcat
+    apiserver: "true"
+spec:
+  version: v1
+  group: appcat.vshn.io
```

Comment:
So I had to double check this, but this will cause some weird issues when we deploy it.

This `APIService` definition basically tells the k8s API server that we as an aggregated API server are responsible for this api group. Than means if a user requests a resource `foo.appcat.io` the k8s API server will basically just blindly forward the request to us.

That's a problem for `objectbuckets.appcat.vshn.io `. If we install this, we are unable to interact with any objectbuckets. We need to have a unique API group for the types provided by the aggregated API server

Reply:
so we need to have a different group for our API server. We might call it `appcat.service.vshn.io`.

Reply:
Exactly. `appcat.apiservice.vshn.io` or `apiservice.appcat.vshn.io` or `api.appcat.vshn.io` or IDK naming is hard, but it needs to be unique

Reply:
Maybe @Kidswiss or @tobru has some input.

Reply:
I'd go with `api.appcat.vshn.io`.

But let's also get tobru's input on that. 👍

Reply:
Changed to `api.appcat.vshn.io`